spec/rubocop/cop/bugcrowd/sidekiq_testing_inline_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Bugcrowd::SidekiqTestingInline do
+  subject(:cop) { described_class.new(config) }
+
+  let(:config) { RuboCop::Config.new }
+
+  it 'registers an offence when calling Sidekiq::Testing::Inline!' do
+    expect_offense(<<~RUBY)
+      Sidekiq::Testing.inline!
```

Comment:
Maybe just examples closer to what it'd look like in real usage. Sometimes writing these specs catches small bugs.

```suggestion
      Sidekiq::Testing.inline! do
        subject
      end
```

and 

```suggestion
      Sidekiq::Testing.inline! { do_something }
```